tools/scripts/brokenLinkChecker/brokenLinkChecker.js

```diff
@@ -3,10 +3,13 @@
 const blc = require('broken-link-checker');
 const https = require('https');
 const url = require('url');
+const YAML = require('yamljs');
 
 const whitelist = require('./whitelist.json').ignore;
 
-const slackUrl = "https://hooks.slack.com/services/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";
+const slackUrl =
+  "https://hooks.slack.com/services/" +
+  YAML.load('../../../globals.yml')['slack_endpoint'];
```

Comment:
[Looks like it logs locally](https://github.com/code-dot-org/code-dot-org/pull/12734/files#diff-d4272021aa27345ff31d42be2245a8ceR39), which we're unlikely to notice IMO.